subscriptions/subscriptions-impl/src/test/java/com/duckduckgo/subscriptions/impl/pixels/SubscriptionPixelTest.kt

```diff
@@ -15,21 +16,24 @@ class SubscriptionPixelTest(
 ) {
     @Test
     fun `pixel name has privacy pro namespace prefix`() {
-        pixel.getPixelNames().values.forEach { pixelName ->
-            assertTrue(pixelName.startsWith("m_privacy-pro_"))
+        SubscriptionPixel.entries.filterNot { it == PURCHASE_SUCCESS_ORIGIN }.forEach {
```

Comment:
nit: This test still lives inside a parameterized test class, but is ignoring the parameter. It will iterate over `SubscriptionPixel.entries` multiple times, potentially making test results noisy in case of a failure. To skip assertion for one particular enum value, I'd just add `if (pixel == PURCHASE_SUCCESS_ORIGIN) return` at the beginning of the method instead of looping through `SubscriptionPixel.entries`. Same goes for another test method below.

Reply:
Yeah didn't saw it was parameterized at all :). Changed!